.github/workflows/publish-release-npm-package.yml

```
@@ -32,12 +31,17 @@ jobs:
 
             - name: 🔖 Parse tag
               id: parse
-              run: echo "pkg=$(dirname $TAG)" >> $GITHUB_OUTPUT
+              run: |
+                  PKG=$(dirname $TAG)
+                  if [[ -d $PKG && "$(cat "$PKG/package.json" | jq -r .private)" != "true" ]]; then
```

From `shellcheck`:
> SC2002 (style): Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.

Also, it's redundant to check whether the `$PKG` directory exists, since `jq` can check whether the `package.json` file exists & is valid.

```suggestion
                  if jq -e '.private != true' "$PKG/package.json" >/dev/null; then
```
With `-e`, `jq` exits successfully if its last output is truthy.  Using that makes it easier to let `jq`'s exit status reflect both the value of `private` and the validity of `package.json`.

---

Oh that's neat, never thought to do boolean comparisons in jq itself, slick.